sentry/content.md

```diff
@@ -49,15 +49,15 @@ Sentry is a realtime event logging and aggregation platform. It specializes in m
 	-	using the celery image:
 
 		```console
-		$ docker run -d --name celery-beat --link some-postgres:postgres --link some-redis:redis  -e CELERY_BROKER_URL=redis://redis celery celery beat
-		$ docker run -d --name celery-worker1  --link some-postgres:postgres --link some-redis:redis  -e CELERY_BROKER_URL=redis://redis celery
+		$ docker run -d --name celery-beat --link some-postgres:postgres --link some-redis:redis -e CELERY_BROKER_URL=redis://redis celery celery beat
```

Comment:
Yeah, then this entire section should be removed since this 100% will not work with the vanilla celery image.

I didn't even realize this was here. :(